src/client/app/components/about.ts

```diff
@@ -41,45 +41,45 @@ export class About implements AfterViewInit {
     @Input() show: boolean;
     @Output() showChange = new EventEmitter<boolean>();
 
+    cache = `
```

Comment:
Using the template strings feels very weird here.  How about just putting these items as three array items, and doing a `.join('\n')`?

    cache = [
        Strings.aboutStorage,
        storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings.aboutSnippets),
        storageSize(sessionStorage, 'playground_intellisense', Strings.aboutIntellisense)
    ].join('\n');